bounce/db/__init__.py

```diff
@@ -19,6 +19,10 @@ def create_engine(driver, user, password, host, port, db_name):
         f'{driver}://{user}:{password}@{host}:{port}/{db_name}', echo=True)
 
 
+def conf_mappers():
```

Comment:
What's this for?

Reply:
Sorry, the conf_mappers() is for searching clubs on the backend, but that should be for the next ticket.  I'll remove it for this PR